apps/api/tests/Animals.spec.ts

```diff
@@ -0,0 +1,201 @@
+import { test, expect, request, APIRequestContext } from '@playwright/test';
+import { defineConfig } from '@playwright/test';
+export default defineConfig({
+  use: {
+    proxy: {
+      server: 'http://local:8080',
+    },
+  }
+});
+import AnimalsListDataSet from '../utils/AnimalsListTestsData.json'
+import NewPetData from '../utils/NewPetDataSet.json'
+import NewPetDataFail from '../utils/NewPetDataSetFail.json'
+import NewPetDataFailEmpty from '../utils/NewPetDataFailEmpty.json'
+import NewPetDataSetFailWrongParameter from '../utils/NewPetDataSetFailWrongParameter.json'
+import NewPetDataUpdate from '../utils/NewPetDataSetUpdate.json'
+
+let url: string;
```

Comment:
```suggestion
const apiUrl: string = "http://localhost:8000/api/v1";
```
This can be a const since it's a fixed string. And you can remove the assignment on line 26. 

Also, WDYT about changing the name to be `apiUrl`? so it's clearer what URL is about?

Reply:
Done, thanks!